web-frontend/cypress/e2e/acceptance/home/send.cy.tsx

```diff
@@ -54,59 +80,222 @@ describe('E2E | Acceptance | Home', () => {
-    //   cy.wrap(
-    //     Cypress.automation('remote:debugger:protocol', {
-    //       command: 'Browser.grantPermissions',
-    //       params: {
-    //         permissions: ['clipboardReadWrite', 'clipboardSanitizedWrite'],
-    //         origin: window.location.origin,
-    //       },
-    //     }),
-    //   );
+    it('should transfer to accounts', () => {
+      const randomIndex = Math.floor(Math.random() * mockedAccounts.length);
+      const selectedAccount = mockedAccounts.at(randomIndex);
+
+      navigateToTransfercoins(2);
+
+      cy.get('[data-testid="transfer-between-accounts"]').click();
+
+      cy.get('[data-testid="popup-modal-content"').should('be.visible');
+
+      for (let i = 0; i < mockedAccounts.length; i++) {
+        cy.get(`[data-testid="selector-account-${i}"]`).should('be.visible');
+      }
+
+      cy.get(`[data-testid="selector-account-${randomIndex}"]`).click();
+      cy.get(`[data-testid="input-field"]`).should(
+        'have.value',
+        selectedAccount.address,
+      );
+    });
+
+    it('should refuse wrong currency input', () => {
+      const account = mockedAccounts.at(2);
+      const recipientAccount = mockedAccounts.at(1);
+
+      const amount = 1000.12311132;
+      const invalidAmount = 'things';
+      const tooMuch = Number(account?.candidateBalance) + 1000;
+      const tooLow = 0;
+      const notEnoughForFees = Number(account?.candidateBalance);
+
+      const standardFees = '1000';
+      navigateToTransfercoins(2);
 
-    //   const account = mockedAccounts.at(2);
+      cy.get('[data-testid="currency-field"')
+        .should('exist')
+        .type(invalidAmount);
 
-    //   cy.visit('/');
+      cy.get('[data-testid="input-field"').type(recipientAccount.address);
 
-    //   cy.get('[data-testid="account-2"]').click();
-    //   cy.url().should('eq', `${baseUrl}/Mario/home`);
+      cy.get('[data-testid="button"]').contains('Send').click();
 
-    //   compareSnapshot(cy, 'wallet-home');
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Invalid amount',
+      );
 
-    //   cy.get('[data-testid="clipboard-field"]').click();
-    //   cy.assertValueCopiedFromClipboard(account.address);
-    // });
+      cy.get('[data-testid="currency-field"').clear().type(tooMuch);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
+
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Insufficient funds',
+      );
+
+      cy.get('[data-testid="currency-field"').clear().type(tooLow);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
+
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Amount must be greater than zero',
+      );
+
+      cy.get('[data-testid="currency-field"').clear().type(notEnoughForFees);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
```

Comment:
Please, let's keep the rule: block actions then asserts.
These two lines are part of an action block: clear the field then click. So, put them together. Then, the assert of that in a new line.